omniduct/filesystems/s3.py

```diff
@@ -189,11 +191,73 @@ def _dir(self, path):
     #         if pattern is None or pattern.match(k.key[len(path_prefix):]):
     #             yield k.key
 
+    def _format_s3_path(self, path):
```

Comment:
Why do we need this? (All filesystems at this point use paths relative to their root, in this case, the bucket name; this doesn't follow that pattern.) If this is necessary for some reason, it should be incorporated into `_path`, but my sense is that it is not. Happy to be wrong.